src/utils/chatService.js

```diff
@@ -26,16 +29,43 @@ export default class ChatService {
   }
 
   send = (message) => {
-    console.log("mardita sea el guevo", message);
     this.service.sendMessage(message);
   }
 
   onNewMessage = () => {
     this.event.addListener('newMessage', ((message) => {
-      console.log(message);
+      const parse = JSON.parse(message);
```

Comment:
```suggestion
      const parse = JSON.parse(message.contents);
```

Reply:
I think you are sending unnecessary information to the javaScript it should arrive filtered from the java.

Reply:
It can be fixed then on ChatServiceModule.java where the event is generated. I don't know if any other type can be used instead of a `WritableMap`